content/gophercon-2016/gophercon2016-retrospective.md

```diff
@@ -0,0 +1,50 @@
+
+Finally, we’d like to thank all of our sponsors.  Some of them, like [Google](http://www.google.com), [Twitch](http://twitch.tv), [Ardan](http://www.ardanlabs.com), [Digital Ocean](http://www.digitalocean.com), [Apcera](http://www.apcera.com), and [Iron.io](http://iron.io) have sponsored all three years.  Words can’t express our gratitude for your constant support.  Other sponsors are making their first appearance at GopherCon, and it was fun to meet them and see what they’re bringing to the Go community.  Here’s a list of this year’s sponsors.  Do us and them a favor and think of them first when you’re searching for a vendor or a place to do business.  They help us keep GopherCon one of the most affordable large conferences out there, and that benefits all of us.
+
+
+
+|Diamond &nbsp;|Platinum &nbsp;|Gold &nbsp;|Silver &nbsp;|Bronze &nbsp;|Other &nbsp;|
+|---|---|---|---|---|---|
+|[Google](http://www.google.com) | [Twitch](http://twitch.tv)  | [Minio](https://minio.io)  |  [Iron.io](http://iron.io) | [CrowdStrike](http://www.crowdstrike.com)  | [Rollbar](http://www.rollbar.com)  |
+|   | [Ardan Labs](http://www.ardanlabs.com)  | [Datadog](http://www.datadoghq.com)  | [Hashicorp](http://www.hashicorp.com) &nbsp; | [Docker](http://www.docker.io)  | [Ticketmaster](http://www.ticketmaster.com)  |
+|   | [Compose.io](http://compose.io)  | [CoreOS](http://www.coreos.com)  | [Comcast](http://www.comcast.com)  |  [Apcera](http://www.apcera.com) | [Backtrace](http://www.backtrace.io)  |
+|   | [Verizon](http://www.verizon.com) | [The Voleon Group](http://voleon.com) &nbsp; | [InfluxData](http://www.influxdata.com)  | [Intel](http://www.intel.com)  |   |
+|   | [Digital Ocean](http://www.digitalocean.com) &nbsp;| [Pivotal](http://www.pivotal.io)  |  [Heroku](http://www.heroku.com) | [LinkedIn](http://www.linkedin.com)  |   |
+|   |   | [New Relic](http://www.newrelic.com)  | [YikYak](http://www.yikyakapp.com)  | [Netflix](http://www.netflix.com)  |   |
+|   |   |   |  [backplane](http://backplane.io) | [RedVentures](http://www.redventures.com) &nbsp; |   |
+|   | |   |   | [Convox](http://convox.com)  |   |
+|   |  |   |   |  |   |
+
+
+At the end of the day, a conference will be judged by the experience at the event and by the speakers it puts on the stage.  We think this year’s speakers were the best we’ve ever had.  Thank you to [Dave Cheney](http://www.twitter.com/davecheney) for curating our program this year and making sure that first-time speakers had a mentor to help them bring their best talk to the big stage.  We were blown away by the professionalism and preparation of the speakers this year.  You can see the slides [here](http://github.com/gophercon/2016-talks), and we’ll update that repository with links to the videos when they’re ready.  
```

Comment:
Can you give a rough date when they might be ready, even if its vague; ie "by the end of next month". or something.